plugins/modules/system/seport.py

```diff
@@ -131,7 +144,7 @@ def semanage_port_get_ports(seport, setype, proto):
     :rtype: list
     :return: List of ports that have the specified SELinux type.
     """
-    records = seport.get_all_by_type()
+    records = seport.get_all_by_type(locallist=local)
```

Comment:
Is this option supported by all versions of the used libraries, or does it require certain version(s)?

Reply:
It's supported since the initial import from svn to git in 2008 and most likely even longer.